packages/node_modules/nav-frontend-js-utils/src/validator.js

```diff
@@ -0,0 +1,119 @@
+import { isIE, mod11OfNumberWithControlDigit } from './';
+
+export class Validator { // eslint-disable-line import/prefer-default-export
```

Comment:
Norsk VS engelsk kan være en diskusjon som tas for dette prosjektet tror jeg. :)
Er litt blanding fra før av, men vi bør nok i større grad bruke ett språk i kodebasen.

Reply:
Jepp, dette må diskuteres - i prosjektet jeg sitter i skriver vi domenespesifikke-ting på norsk (Person, Arbeid, Stønad etc.), men engelsk på alt annet, som jeg syns føles naturlig. Men en ting vi må være bevisste på om vi skal vurdere engelsk kodebase for nav-frontend er at designerne bruker norsk på alt i designsystemet i Sketch per idag.

Om vi må ha samme begrepsbruk på tvers av de rollene er det kanskje en mulighet å navngi komponenter og attributter som er definert i designsystemet på norsk, og bruke engelsk i kodebasen ellers.